sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py

```diff
@@ -273,29 +565,19 @@ def test_create_service_protocol(self):
             assert service.scheme == "http"
 
     def test_create_service_empty_key(self):
-        for client, url in SERVICES.items():
+        for client in SERVICES:
             # Act
             with pytest.raises(ValueError) as e:
-                test_service = client("testaccount", credential="", table_name="foo", endpoint_type=url)
+                test_service = client("testaccount", credential="", table_name="foo")
 
             assert str(e.value) == "You need to provide either an AzureSasCredential or AzureNamedKeyCredential"
 
     def test_create_service_with_socket_timeout(self):
-        for client, url in SERVICES.items():
+        url = self.account_url(self.tables_cosmos_account_name, "cosmos")
+        for client in SERVICES:
             # Act
-            default_service = client(
-                endpoint=self.account_url(self.tables_cosmos_account_name, url),
-                credential=self.credential,
-                table_name="foo",
-                endpoint_type=url,
-            )
-            service = client(
-                endpoint=self.account_url(self.tables_cosmos_account_name, url),
-                credential=self.credential,
-                table_name="foo",
-                connection_timeout=22,
-                endpoint_type=url,
```

Comment:
Looks like we were actually testing the `endpoint_type` parameter. 
That's interesting.... I'd say in that case we shouldn't remove it.

Reply:
I see you've removed all use of `endpoint_type` from the tests - did doing that actually change or break the tests in any way? Did you have to "fix" anything?

Reply:
All the tests with `endpoint_type` are unit tests. I don't see any behavior change after removing it.